src/app/support/components/custom-select/custom-select.component.scss

```diff
@@ -0,0 +1,12 @@
+.support {
```

Comment:
It's a little bit strange to be honest. You mixed BEM methodology with SCSS features.